lib/parity/backup.rb

```diff
@@ -53,10 +53,15 @@ def download_remote_backup
     def restore_from_local_temp_backup
       Kernel.system(
         "pg_restore tmp/latest.backup --verbose --clean --no-acl --no-owner "\
-          "-d #{development_db} #{additional_args}",
+          "--dbname #{development_db} --jobs `#{number_of_cores}` "\
+          "#{additional_args}",
       )
     end
 
+    def number_of_cores
+      "getconf _NPROCESSORS_ONLN"
+    end
```

Comment:
I like having Parity do more work and the end-user do less work, so I am more in favor of an if/else depending on the host OS. I am less in favor of making people export a variable to use Parity.